### PR TITLE
SPCTR-3347 Improvement: Buttons - Added a new vertical alignment option.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3347

### Screenshots

- https://d.pr/v/0P138v

### Types of changes

- New feature (non-breaking change which adds functionality)

### How has this been tested?

- Added to the card

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/buttons/styling.js [hunk 1]
```diff
@@ -73,12 +73,15 @@ function styling( attributes, clientId, name, deviceType ) {
...
 		fontLetterSpacingType,
+		verticalAlignment,
 	} = attributes;
 
 	const selectors = {};
 	const tabletSelectors = {};
 	const mobileSelectors = {};
 
+	const vAlign = verticalAlignment === 'top' ? 'flex-start' : verticalAlignment === 'bottom' ? 'flex-end' : 'center'; // eslint-disable-line no-nested-ternary
+
```
Comment (bsf-zanev) [inline src/blocks/buttons/styling.js:84]: Can we `let vAlign` and set it through a switch case on `verticalAlignment`?

File: src/blocks/buttons/styling.js [hunk 2]
```diff
@@ -177,9 +180,13 @@ function styling( attributes, clientId, name, deviceType ) {
 		selectors[ '.uagb-editor-preview-mode-desktop .block-editor-inner-blocks' ] = {
 			'text-align': align,
 		};
+		selectors['.uagb-editor-preview-mode-desktop .block-editor-block-list__layout'] = {
```
Comment (bsf-zanev) [inline src/blocks/buttons/styling.js:183]: For consistency, let's keep spacing in all the new lines added to this file, like so:

```suggestion
		selectors[ '.uagb-editor-preview-mode-desktop .block-editor-block-list__layout' ] = {
```

**Review by bsf-zanev — COMMENTED** [review]
A few minor comments.

File: src/blocks/buttons/styling.js (hunk 2 above)
Comment (AbhijitNage123) [reply src/blocks/buttons/styling.js]: Done

File: src/blocks/buttons/styling.js (hunk 1 above)
Comment (AbhijitNage123) [reply src/blocks/buttons/styling.js]: Done

File: includes/blocks/buttons/frontend.css.php [hunk 3]
```diff
@@ -69,18 +69,17 @@
 		'column-gap' => UAGB_Helper::get_css_value( $attr['gapMobile'], 'px' ),
 	);
 }
-$alignment       = ( 'left' === $attr['align'] ) ? 'flex-start' : ( ( 'right' === $attr['align'] ) ? 'flex-end' : 'center' );
-$alignmentTablet = ( 'left' === $attr['alignTablet'] ) ? 'flex-start' : ( ( 'right' === $attr['alignTablet'] ) ? 'flex-end' : 'center' );
-$alignmentMobile = ( 'left' === $attr['alignMobile'] ) ? 'flex-start' : ( ( 'right' === $attr['alignMobile'] ) ? 'flex-end' : 'center' );
 
+$vAlign = ( 'top' === $attr['verticalAlignment'] ) ? 'flex-start' : ( ( 'bottom' === $attr['verticalAlignment'] ) ? 'flex-end' : 'center' );
```
[the remark this answers is not in the document]
Comment (AbhijitNage123) [reply includes/blocks/buttons/frontend.css.php]: Done

**Review by rajmohanreddy774 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Pull request merged**